File: C++/freeCodeCamp.org_C++ Tutorial for Beginners/src/005_Working with Strings/WorkingWithStrings.cpp
#include <iostream>

/* Simple C++ program that uses string functions */

using namespace std;

int main()
{
    string phrase = "Giraffe Academy";
    int phraseLength = phrase.length();
    char phraseCharacter = phrase[0];
    int phraseIndex = phrase.find("Academy", 0);
    string phrasesub = phrase.substr(8,3);

    return 0;
}

File: Python/002_OOP Basics/005_Working with Strings/WorkingWithStrings.py
# This showcases multiple string functions that come in handy and prints out the return values
phrase = "Giraffe Academy";

phraseLength = len(phrase);
print("Phrase Length: ", phraseLength)

phraseCharacter = phrase[0];
print("Phrase Character: ", phraseCharacter)

phraseIndex = phrase.find("Academy", 0);
print("Phrase Index: ",phraseIndex)

phraseSub = phrase[3:8];
print("Phrase Phrase Substring: ", phraseSub)
